src/Core/functions.php

```diff
@@ -179,7 +179,7 @@ function pj($var)
         }
 
         $template = (PHP_SAPI !== 'cli' && PHP_SAPI !== 'phpdbg') ? '<pre class="pj">%s</pre>' : "\n%s\n\n";
-        printf($template, trim(json_encode($var, JSON_PRETTY_PRINT)));
+        printf($template, trim(json_encode($var, JSON_PRETTY_PRINT|JSON_UNESCAPED_UNICODE)));
```

Comment:
```suggestion
        printf($template, trim(json_encode($var, JSON_PRETTY_PRINT|JSON_UNESCAPED_UNICODE|JSON_UNESCAPED_SLASHES)));
```

URLs in JSON are now like `"https:\/\/github.com\/cakephp\/cakephp"`, better without escaping too.

Reply:
ok